testing/test_dataset.py

```diff
@@ -1736,8 +1736,8 @@ def testConstructor(self):
         self.assertEqual(self.eventArray1.session, self.dataset.sessions[0])
         self.assertEqual(self.eventArray1.subchannelId, None)
 
-        self.assertEqual(self.eventArray1._blockIndicesArray.size, 0)
-        self.assertEqual(self.eventArray1._blockTimesArray.size, 0)
+        # self.assertEqual(self.eventArray1._blockIndicesArray.size, 0)
+        # self.assertEqual(self.eventArray1._blockTimesArray.size, 0)
```

Comment:
what happened here?

Reply:
Oops, accidental. This is related to #22. I previously removed the `EventArray._getBlockIndexWith*()` methods, which used those attributes, as they were profiled as being somewhat slower than the old `EventList` versions (there was even a comment speculating they might be). However, removing those two methods revealed a different bug, so I restored the methods for the time being. I just left in the change in the test by accident.

Reply:
gotcha. I see the re-add too, thanks